<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/bird.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="apptitle">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Finite Element Application</string>
    </property>
   </widget>
   <widget class="QLabel" name="modellabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Arial&quot;;
background-color: rgb(170, 255, 255);
color: rgb(255, 85, 0);</string>
    </property>
    <property name="text">
     <string>选择模型</string>
    </property>
   </widget>
   <widget class="QComboBox" name="modelBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>矩形</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>其他</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="loadbtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
    </property>
    <property name="text">
     <string>点击加载</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="loadbox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>461</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QFrame" name="modelimg">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>211</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/mod.jpg);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="n0xline">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>330</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pureback.webp);</string>
     </property>
    </widget>
    <widget class="QLabel" name="n0x">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;
background-image: url(:/pureback.webp);
</string>
     </property>
     <property name="text">
      <string>Fx=</string>
     </property>
    </widget>
    <widget class="QLabel" name="n0y">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pureback.webp);
font: 75 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Fy=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="n0yline">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>380</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pureback.webp);</string>
     </property>
    </widget>
    <widget class="QLabel" name="n1x">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>330</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;
background-image: url(:/pureback.webp);</string>
     </property>
     <property name="text">
      <string>Fx=</string>
     </property>
    </widget>
    <widget class="QLabel" name="n1y">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>380</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;
background-image: url(:/pureback.webp);</string>
     </property>
     <property name="text">
      <string>Fy=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="n1xline">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>330</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pureback.webp);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="n1yline">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>380</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pureback.webp);</string>
     </property>
    </widget>
    <widget class="QLabel" name="n2x">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pureback.webp);
font: 75 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Fx=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="n2xline">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pureback.webp);</string>
     </property>
    </widget>
    <widget class="QLabel" name="n2y">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pureback.webp);
font: 75 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Fy=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="n2yline">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pureback.webp);</string>
     </property>
    </widget>
    <widget class="QLabel" name="n3x">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pureback.webp);
font: 75 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Fx=</string>
     </property>
    </widget>
    <widget class="QLabel" name="n3y">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;
background-image: url(:/pureback.webp);</string>
     </property>
     <property name="text">
      <string>Fy=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="n3xline">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pureback.webp);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="n3yline">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pureback.webp);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Eline">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pureback.webp);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="vline">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pureback.webp);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tline">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>220</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pureback.webp);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="resultbox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>451</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="resulttip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>分析结果如下:</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="resultpage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>451</width>
       <height>261</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page0">
      <attribute name="title">
       <string/>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>单元0应变大小：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>单元0应力大小：</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="strain0">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>301</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="stress0">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>301</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page1">
      <attribute name="title">
       <string/>
      </attribute>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>单元1应变大小：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>单元1应力大小：</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="strain1">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>301</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="stress1">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>301</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="analysebtn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>520</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
    </property>
    <property name="text">
     <string>开始分析</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitbtn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>520</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="author">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>600</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Ink Free&quot;;</string>
    </property>
    <property name="text">
     <string>Created by Zihao-Cai @20221211</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
